src/visualizePath.js

```diff
@@ -0,0 +1,39 @@
+import { PIXELS_PER_TILE } from './constants';
+
+/* eslint no-param-reassign: ["error", { "ignorePropertyModificationsFor": ["renderer"] }] */
+function createPathSprites(path, renderer, cpt) {
+  renderer.pathSprites = [];
+  const pixelsPerCell = PIXELS_PER_TILE / cpt;
+  path.forEach(cell => {
+    // note: all units in pixels
+    const rect = new PIXI.Graphics();
+    rect.beginFill(0x00ff00).drawRect(
+      cell.x * pixelsPerCell, // x coordinate
+      cell.y * pixelsPerCell, // y coordinate
+      pixelsPerCell, // width
+      pixelsPerCell, // height
+    ).alpha = 0.25;
+    renderer.pathSprites.push(rect);
+  });
+}
+
+function clearRects(renderer) {
```

Comment:
A rect is closer to a `cell`, since when our path is more granular, each rect being drawn will correspond to a cell. Good question, will clarify via comment